CODIGO/ModUtils.bas

```diff
@@ -1718,21 +1718,21 @@ Public Sub DibujarMiniMapa()
     
         On Error GoTo DibujarMiniMapa_Err
 
-100     frmMain.MiniMap.Picture = LoadMinimap(UserMap)
+100     frmMain.MiniMap.Picture = LoadMinimap(ResourceMap)
```

Comment:
We need to take the map as an argument rather than using a global `DibujarMiniMapa( byval map as integer)`

Reply:
I agree but this is not related to this change